axon-server-connector/src/main/java/org/axonframework/axonserver/connector/AxonServerConfiguration.java

```diff
@@ -151,6 +151,8 @@ public class AxonServerConfiguration {
      */
     private FlowControlConfiguration commandFlowControl;
 
+    private boolean localSegmentShortCut = false;
```

Comment:
By adding JavaDoc to the variable, an IDE can provide that documentation right away when traversing a `application.properties`/`application.yml` file. So, adding that would be important, and helpful!